github_run.py

```diff
@@ -118,17 +118,14 @@ def new_pr(self, not_posted):
             metadata = yaml.safe_load(metadata)
             if media:
                 metadata["media"] = media
-                metadata["mentions"] = {
-                    key: value
-                    for key, value in metadata.get("mentions", {}).items()
-                    if key in media
-                }
-                metadata["hashtags"] = {
-                    key: value
-                    for key, value in metadata.get("hashtags", {}).items()
-                    if key in media
-                }
-            new_md_content = f"---\n{yaml.dump(metadata)}---\n{text}"
+                for meta in ["mentions", "hashtags"]:
+                    if meta in metadata:
```

Comment:
this if is not needed I think you have `metadata.get(meta, {})` in place

Reply:
This is to prevent the mentions and hashtags to create if they are empty.

Reply:
ok, then the get() is not needed, see below